apps/src/code-studio/components/Libraries/LibraryCreationDialog.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import BaseDialog from '../../../templates/BaseDialog';
+import {connect} from 'react-redux';
+import {hideLibraryCreationDialog} from './libraryCreationRedux';
+import libraryParser from './libraryParser';
+import LibraryClientApi from './LibraryClientApi';
+import i18n from '@cdo/locale';
+
+class LibraryCreationDialog extends React.Component {
```

Comment:
No tests for this yet because none of it is in its final state. This is just scaffolding to give the redux components an endpoint